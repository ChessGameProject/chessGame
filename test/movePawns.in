setup
done
game human human
move a2 a4
move b7 b5
move b2 b3
move a7 a6
move b3 b4
move a6 a5
move a4 b5
move a5 b4
resign
