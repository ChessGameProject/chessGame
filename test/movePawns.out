Welcome to Pawn Pusher 9000!
Commands: 'setup', 'game [white-player] [black-player]'
Welcome to SETUP mode
Commands: '+ [Piece] [Location]', '- [Location]', '= [Player]', 'done'
8 rnbqkbnr
7 pppppppp
6 _ _ _ _
5  _ _ _ _
4 _ _ _ _
3  _ _ _ _
2 PPPPPPPP
1 RNBQKBNR
  abcdefgh
Commands: 'setup', 'game [white-player] [black-player]'
1's turn:
8 rnbqkbnr
7 pppppppp
6 _ _ _ _
5  _ _ _ _
4 P _ _ _
3  _ _ _ _
2 _PPPPPPP
1 RNBQKBNR
  abcdefgh
-1's turn:
8 rnbqkbnr
7 p_pppppp
6 _ _ _ _
5  p _ _ _
4 P _ _ _
3  _ _ _ _
2 _PPPPPPP
1 RNBQKBNR
  abcdefgh
1's turn:
8 rnbqkbnr
7 p_pppppp
6 _ _ _ _
5  p _ _ _
4 P _ _ _
3  P _ _ _
2 _ PPPPPP
1 RNBQKBNR
  abcdefgh
-1's turn:
8 rnbqkbnr
7  _pppppp
6 p _ _ _
5  p _ _ _
4 P _ _ _
3  P _ _ _
2 _ PPPPPP
1 RNBQKBNR
  abcdefgh
1's turn:
8 rnbqkbnr
7  _pppppp
6 p _ _ _
5  p _ _ _
4 PP_ _ _
3  _ _ _ _
2 _ PPPPPP
1 RNBQKBNR
  abcdefgh
-1's turn:
8 rnbqkbnr
7  _pppppp
6 _ _ _ _
5 pp _ _ _
4 PP_ _ _
3  _ _ _ _
2 _ PPPPPP
1 RNBQKBNR
  abcdefgh
1's turn:
8 rnbqkbnr
7  _pppppp
6 _ _ _ _
5 pP _ _ _
4 _P_ _ _
3  _ _ _ _
2 _ PPPPPP
1 RNBQKBNR
  abcdefgh
-1's turn:
8 rnbqkbnr
7  _pppppp
6 _ _ _ _
5  P _ _ _
4 _p_ _ _
3  _ _ _ _
2 _ PPPPPP
1 RNBQKBNR
  abcdefgh
1's turn:
Commands: 'setup', 'game [white-player] [black-player]'
